apps/github-cascading-app/schemas/config.schema.json

```diff
@@ -31,6 +31,11 @@
       "description": "Pattern determining if the branch is part of the cascading strategy",
       "default": "^releases?/\\d+\\.\\d+"
     },
+    "excludeBranchesWithHigherMinor": {
```

Comment:
I find this name very confusing, it seems that it's actually doing the exact opposite `onlyCascadeOnHighestMinors`